openhands/events/observation/condenser.py

```diff
@@ -0,0 +1,15 @@
+from dataclasses import dataclass
+
+from openhands.core.schema import ObservationType
+from openhands.events.observation.observation import Observation
+
+
+@dataclass
+class CondensationObservation(Observation):
```

Comment:
Just a small thing: could we do this in agent.py ? Just because there's an old AgentSummarizeAction in the `action` counterpart, which seems like the same kind of thing.

It may also be nice to name it AgentCondensationObservation, just for the naming pattern consistency.

Sorry, it's just that it's a little tricky to move/rename later, because they're events and we try to keep compatibility with old streams...